app/src/test/java/org/mozilla/tv/firefox/architecture/FirefoxViewModelProvidersTest.kt

```diff
@@ -29,11 +27,6 @@ class FirefoxViewModelProvidersTest {
     fun `WHEN passed an unknown ViewModel THEN of(FragmentActivity) throws an exception`() {
         FirefoxViewModelProviders.of(mainActivity).get(UnknownViewModel::class.java)
     }
-
-    @Test
-    fun `WHEN passed a valid ViewModel THEN of(FragmentActivity) returns a non-null value`() {
-        assertNotNull(FirefoxViewModelProviders.of(mainActivity).get(PocketViewModel::class.java))
```

Comment:
You should switch this to a non-Pocket view model because the test is still valid otherwise